### PR TITLE
UI issues with large font fix issue#28

### PR DESCRIPTION
**Pull request by neeldoshii**
Linked Issue : #28 

## Testing Instructions
1. Change your device font size to large
2. Go to settings -> Then Go to Help and Feedback

## Screenshots or Screencast 
Screenshot 2023-09-26 at 4 24 14 PM

## Checklist
- [ ] If this is a user-facing change, I have added an entry in CHANGELOG.md
- [ ] Ensure the linter passes (`./gradlew spotlessApply` to automatically apply formatting/linting)
- [ ] I have considered whether it makes sense to add tests for my changes
- [ ] All strings that need to be localized are in `modules/services/localization/src/main/res/values/strings.xml`
- [ ] Any jetpack compose components I added or changed are covered by compose previews
 
#### I have tested any UI changes...

- [ ] with different themes
- [ ] with a landscape orientation
- [X] with the device set to have a large display and font size
- [ ] for accessibility with TalkBack

## Fixed
- [X]  Help & Feedback Screen has a bit of horizontal scroll
- [X] Help & Feedback "Still stuck..." box cuts words in half
- [X] Help & Feedback "Get in touch popup" has buttons outside the popup
- [ ] Not all toolbars reflect the updated font size. 
- [ ]  Profile's Active Account section loses its top and bottom padding
- [ ] Podcast page's folder icon feels a bit crowded against the podcast picture
- [ ] The Choose Folder Color screen's colors crowd the right side of the screen

The tick indicates the fixed rest is not fixed. This PR is partial bug fixes.

### CODE REVIEW

File: modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/HelpFragment.kt [hunk 1]
```diff
@@ -90,16 +101,29 @@ class HelpFragment : BaseFragment(), HasBackstack, Toolbar.OnMenuItemClickListen
...
             webViewClient = SupportWebViewClient()
             loadUrl(loadedUrl ?: Settings.INFO_FAQ_URL)
             settings.javaScriptEnabled = true
             settings.textZoom = 100
+            //disables horizontal scrolling bar not scroll
+            isHorizontalScrollBarEnabled = false
+            settings.layoutAlgorithm = TEXT_AUTOSIZING
             settings.domStorageEnabled = true
+            // Will allow wide viewport as much device width has for responsiveness
+            settings.useWideViewPort = true
```
Comment (mchowning) [inline modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/HelpFragment.kt:121]: I wasn't able to tell what difference this made on the Help & Feedback screen. Can help me understand why this is needed?

File: modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/HelpFragment.kt [hunk 2]
```diff
@@ -90,16 +101,29 @@ class HelpFragment : BaseFragment(), HasBackstack, Toolbar.OnMenuItemClickListen
...
         val view = inflater.inflate(VR.layout.fragment_webview, container, false)
 
         webView = (view.findViewById<View>(VR.id.webview) as WebView).apply {
             webViewClient = SupportWebViewClient()
             loadUrl(loadedUrl ?: Settings.INFO_FAQ_URL)
             settings.javaScriptEnabled = true
             settings.textZoom = 100
+            //disables horizontal scrolling bar not scroll
+            isHorizontalScrollBarEnabled = false
+            settings.layoutAlgorithm = TEXT_AUTOSIZING
```
Comment (mchowning) [inline modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/HelpFragment.kt:118]: I wasn't able to tell what difference changing the layoutAlgorithm made. Can help me understand why we need to set this?

**Review by mchowning — CHANGES_REQUESTED** [review]
Thanks for the contrubution @neeldoshii and sorry it's taken awhile to get it reviewed. 

I left a couple of comments just clarifying why we needed a few changes. I didn't see any issues from those changes, but I just want to understand why we need them (and make sure they are necessary changes).

The only significant issue I noticed is that I'm seeing an extra black bar across the top of the screen that wasn't there before. It looks like something with the padding is off. It's a little hard to see in this screenshot, but if you look closely, you can see there is a black bar below the almost-black toolbar.

Screenshot 2023-10-18 at 3 03 38 PM

Also there are some linter errors. Running the spotlessApply gradle task should fix those.

**Comment by neeldoshii** [comment]
I am closing this PR as this code is not properly optimized. Fixing a small issue is creating 2-3 more issue.
I think it needs much better approach to tackle this the time I wrote that code I was very beginner.

**Pull request closed without merging**